firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteIndexManager.java

```diff
@@ -233,6 +239,21 @@ public void deleteFieldIndex(FieldIndex index) {
     }
   }
 
+  @Override
+  public void createTargetIndices(Target target) {
+    hardAssert(started, "IndexManager not started");
+
+    if (!autoClientIndexingEnabled) return;
+
+    for (Target subTarget : getSubTargets(target)) {
+      IndexType type = getIndexType(subTarget);
+      if (type == IndexType.NONE) {
```

Comment:
This can be None or Partial? When we decide to build index, we can always make sure it's a full index.